maqs-utilities/src/main/java/com/cognizantsoftvision/maqs/utilities/logging/IHtmlFileLogger.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2022 (C) Cognizant SoftVision, All rights Reserved
+ */
+
+package com.cognizantsoftvision.maqs.utilities.logging;
+
+/**
+ * The Html File Logger interface class.
+ */
+public interface IHtmlFileLogger extends AutoCloseable {
```

Comment:
Weird name for this interface.
I know we use if for html, but it is more generic than that.
IAutoCloseableLogger seems like a better name

Reply:
deleted this interface as IFileLogger extension works